composer.json

```diff
@@ -205,7 +211,11 @@
     "su-sws/stanford_media": "dev-8.x-1.x",
     "su-sws/stanford_ssp": "dev-8.x-1.x"
   },
+  "replace": {
+    "dealerdirect/phpcodesniffer-composer-installer": "*"
```

Comment:
Perhaps Greg or I should take a pass at this issue. This seems a tad risky.

Reply:
I could not for the life of me get this to work. Even patching the library didnt work. I also don't think it belongs in `dependencies`. If anything BLT should require it as a `devDependency` IMO. All this library does is automatically install phpcs standards when a dependency changes/added/removed. Not really needed.